src/dapp/DSRuneAbstract.sol

```diff
@@ -13,7 +13,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-// SPDX-License-Identifier: GPL-3.0-or-later
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
Preexisting issue here, but SPDX header should probably be moved above the copyright block.

Reply:
Fixed.